src/test/java/org/humancellatlas/ingest/project/ProjectServiceTest.java

```diff
@@ -287,4 +293,133 @@ void givenSuggestionCreatesProjectSuccessfully() {
             assertThat(result).isEqualTo(persistentProject);
         }
     }
+
+
+    @Nested
+    class FilterProject {
+        private String primaryWranglerKey = "primaryWrangler";
+        private String isUpdateKey = "isUpdate";
+        private String wranglingStateKey = "wranglingState";
+        private String projectTitleKey = "content.project_core.project_title";
+        private String projectDescripionKey = "content.project_core.project_description";
+        private String projectShortNameKey = "content.project_core.project_short_name";
+
+        private String searchText = "search text";
+        private String wranglingStateValue = "NEW";
+        private String primaryWranglerValue = "primary wrangler 1";
+
+        Criteria getDefaultFilter() {
+            return Criteria.where(isUpdateKey).is(false);
+        }
+
+        Criteria filterByWrangler(String wrangler) {
+            return Criteria.where(primaryWranglerKey).is(wrangler);
+        }
+
+        Criteria filterByWranglingState(String wranglingState) {
+            return Criteria.where(wranglingStateKey).is(wranglingState);
+        }
+
+        Criteria filterBySearch(String search) {
```

Comment:
This is a repetition of what happens in the actual code.
The test needs to be simpler than the code, otherwise, how can you make sure that the test is correct?